lib/pluginmanager/package.rb

```diff
@@ -0,0 +1,67 @@
+class LogStash::PluginManager::Package < LogStash::PluginManager::Command
+
+  option "--tgz", :flag, "compress package as a tar.gz file", :default => false
+  option "--zip", :flag, "compress package as a zip file", :default => false
+  option "--clean", :flag, "clean up the generated dump of plugins", :default => false
+  option "--override", :flag, "Override a previously generated package file", :default => false
```

Comment:
I added with the intention to simplify life for things like chef and puppet, so they could do override and forget about the rest. I wanted to reduce the amount of hassle involved if possible. This is minor to me, but I would keep it.